### PR TITLE
Test/868 staking ACs

### LINKED ISSUE #868: Add AC references to staking tests
We want to start integrating ACs with Cypress tests.

This may require tests to be rewritten or new tests added to fit with the ACs

ACs can be found in:
https://github.com/vegaprotocol/specs/blob/Add-interface-acceptance-criteria/user-interface/1002-STKE-staking.md

For now we can reference ACs by adding comments next to tests with the AC ids

Tool for measuring coverage:

### PR DESCRIPTION
**Pull request by jtsang586**
# Related issues 🔗

Closes #868 & #884 

# Description ℹ️

Adds some additional tests and ACs. Fix broken flow tests from the validator list changes.

### CODE REVIEW

File: apps/token-e2e/src/integration/flow/staking-flow.cy.js [hunk 1]
```diff
@@ -435,7 +431,7 @@ context.skip('Staking Tab - with eth and vega wallets connected', function () {
 
       cy.get(vegaWalletStakedBalances, txTimeout)
         .should('contain', 2.0, txTimeout)
-        .and('contain', this.validatorName);
+        .and('contain', '');
```
Comment (sam-keen) [inline apps/token-e2e/src/integration/flow/staking-flow.cy.js:434]: Question (non-blocking): why check for an empty string here?

**Review by sam-keen — APPROVED** [review]

File: apps/token-e2e/src/integration/flow/staking-flow.cy.js (hunk 1 above)
Comment (jtsang586) [reply apps/token-e2e/src/integration/flow/staking-flow.cy.js]: Nice spot. Corrected!

File: apps/token-e2e/src/integration/flow/staking-flow.cy.js [hunk 2]
```diff
@@ -199,22 +199,25 @@ context.skip('Staking Tab - with eth and vega wallets connected', function () {
...
 
+      // 1002-STKE-049
       cy.get(stakeNextEpochValue, epochTimeout)
         .contains(2.0, epochTimeout)
         .should('be.visible');
 
       cy.get(vegaWalletNextEpochBalances, txTimeout)
         .should('contain', 2.0, txTimeout)
-        .and('contain', this.validatorName)
+        .and('contain', partValidatorId)
```
Comment (mattrussell36) [inline apps/token-e2e/src/integration/flow/staking-flow.cy.js:215]: question (non blocking): Seems a odd to be asserting that the ellipsis is displayed in so many places? I might be missing something here.

**Review by mattrussell36 — APPROVED** [review]

File: apps/token-e2e/src/integration/flow/staking-flow.cy.js (hunk 2 above)
Comment (jtsang586) [reply apps/token-e2e/src/integration/flow/staking-flow.cy.js]: Originally there were kind of like placeholders for when we could get validators in Capsule with names but that probably won't happen anytime soon. I can also see there's a lot of repeated assertions for the epoch text. I've tidied that up so it's only asserting the texts once per element in the tests.

**Pull request merged**